h2o-py/h2o/utils/shared_utils.py

```diff
@@ -353,6 +358,144 @@ def slice_is_normalized(s):
+
+    :param mojo_model: This is the MOJO model file name which is download after model was build using H2O, you can pass it two ways:
+           1. Full mojo model path i.e. /Users/avkashchauhan/src/github.com/h2oai/h2o-tutorials/tutorials/python_mojo_scoring/gbm_prostate_new.zip
+           2. mojo name first i.e. gbm_prostate_new.zip and genmodelpath = /Users/avkashchauhan/src/github.com/h2oai/h2o-tutorials/tutorials/python_mojo_scoring
+           3. only mojo model name: In this case system uses os.path.abspath to get the folder where mojo is and then look for h2o-genmodel.jar in the same location
+    :param json:  convert pandas frame to json (pd.to_json) and pass here
+    :param genmodelpath: This is the local file system folder name where both h2o-genmodel.jar and MOJO zip file is stored.
+    :param labels:     (Optional) True : Shows results, False: Does no show results and just pass result to given object
+    :param javaoptions: (Optional) These are the Java string options given by the user
+    :param show_debug: True/False - Default ( Use True to see the full Java command line)
+    :return: score as json values
+    """
+    # Verifying JSON
+    if show_debug:
+        print(json)
+
+    check_json(json)
+
+    # Checking java
+    java = H2OLocalServer._find_java()
+
+    jver_bytes = subprocess.check_output([java, "-version"], stderr=subprocess.STDOUT)
+    jver = jver_bytes.decode(encoding="utf-8", errors="ignore")
+    if show_debug:
+        print("  Java Version: " + jver.strip().replace("\n", "; "))
+    if "GNU libgcj" in jver:
+        raise H2OStartupError("Sorry, GNU Java is not supported for H2O.\n"
+                              "Please download the latest 64-bit Java SE JDK from Oracle.")
+    """
+    if "Client VM" in jver:
+        warn("  You have a 32-bit version of Java. H2O works best with 64-bit Java.\n"
+             "  Please download the latest 64-bit Java SE JDK from Oracle.\n")
+    # genmodelpath > must start and ends with "/" or add to it
+
+    if sys.platform == "win32":
+        separator = ";"
+    else:
+        separator = ":"
+    """
+
+    separator = os.pathsep
+    file_separator = os.sep
+
+    gen_model_arg = "." + separator
+
+    ## Working on mojo_model
+    mojo_model_path=None
+
+    head, tail = os.path.split(mojo_model)
+    if not head:
+        if not genmodelpath:
+            # taking path from asbpath, which is based on given mojo model file name
+            head, tail = os.path.split(os.path.abspath(mojo_model))
+            mojo_model_path = head
+        else:
+            # taking path from given genmodelpath
+            mojo_model_path = genmodelpath
+    else:
+        # Taking path from the path given with model mojo file
+        mojo_model_path = head
+
+    if not mojo_model_path:
+        raise RuntimeError("Error: given mojo model " + mojo_model + " does not have full path!")
```

Comment:
Here and below, `ValueError` is more appropriate (i.e. the value given by the user was wrong)